viewer/packages/sector-parser/app/testMaterials.ts

```diff
@@ -0,0 +1,275 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import * as THREE from 'three';
+
+import matCapTextureImage from '../../../core/src/datamodels/cad/rendering/matCapTextureData';
+import { sectorShaders } from '../../../core/src/datamodels/cad/rendering/shaders';
```

Comment:
Importing internals from core seems a bit dodgy, even for a test app. Is it a good idea to separate rendering stuff to a new package, e.g. 'cad-rendering'?

Reply:
Yeah, I agree with this. But was great having this test-app when developing. I agree that importing internals is not ideal, but the alternative makes it quite hard to further develop because it is very valuable to see what is going on. I really don't want to give up the ability to run an app here.

Best approach in my opinion would be to have a package containing shaders, matcaptexture and other essentials that could be imported here. There are downsides to this though, lets discuss in person.

Reply:
Did we decide not to do this?